test/utils.js

```diff
@@ -0,0 +1,70 @@
+import { watchFile, readdirSync, statSync } from 'fs';
```

Comment:
Some inconsistent usage of ES6 in this file.

- `module.exports` should be replaced with [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export), e.g. `export function watch( files, fn ) {`
- A few `var`, to be replaced with `const` or `let` ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#variable-declarations))